fdbcli/BulkLoadCommand.actor.cpp

```diff
@@ -291,6 +291,50 @@ ACTOR Future<UID> bulkLoadCommandActor(Database cx, std::vector<StringRef> token
 		printLongDesc(tokens[0]);
 		return UID();
 
+	} else if (tokencmp(tokens[1], "printlock")) {
+		// For debugging purposes and invisible to users.
+		if (tokens.size() != 2) {
+			fmt::println("{}", BULK_LOAD_STATUS_USAGE);
+			return UID();
+		}
+		std::vector<std::pair<KeyRange, RangeLockState>> lockedRanges =
+		    wait(findExclusiveReadLockOnRange(cx, normalKeys));
+		fmt::println("Total {} locked ranges", std::to_string(lockedRanges.size()));
```

Comment:
Nit: `std::to_string` is not needed. `fmt:pringln` is smart to know how to format the int.

Reply:
Thanks! Changed.